### PR TITLE
Prettify Site Detail, add Edit button

### PR DESCRIPTION
**Pull request by baileysingleton**
fixes issue #301 

## Changes in this PR.

- [X] Changed the layout of the site_detail to be more symmetrical
- [X] Added an edit site button that redirects to the edit site page (which could use some work!)
- [X] Centered Image
- [X] Removed created and modified dates

## Testing this PR.

1. Load up docker
2. Localhost:8000
3. Sites->Select a Site
4. Admire the site detail page
5. Or criticize it, your choice. 

### Expected Output.

```
Site_detail = "looks good"
```

2017-07-19-160718_1079x1919_scrot

@osuosl/devs

### CODE REVIEW

**Comment by Kennric** [comment]
Where does the site description show up?

I think I prefer something closer to the old site, but with a larger photo. Maybe the map and photo next to each other, description below photo, and data/ graphs together below that...

**Comment by Kennric** [comment]
Can you try a few different layouts and get screen caps of them to show Renee?

**Comment by baileysingleton** [comment]
@Kennric 
layout2

Here's another layout, like the one you described. Would you like me to send over some screen shots to Renee, of the two layouts or should there be a third?

File: streamwebs_frontend/streamwebs/templates/streamwebs/site_detail.html [hunk 1]
```diff
@@ -177,6 +143,47 @@
...
+        <li>
+          <a href="{% url 'streamwebs:soil_edit' site_slug=site.site_slug %}">
+            {% trans "Soil Survey" %}
+          </a>
+        </li>
+      </ul>
+      <a href="{% url 'streamwebs:update_site' site_slug=site.site_slug %}"
+        class="btn-large teal darken-4">
+        <i class="material-icons right">mode_edit</i>
+        {% trans "Edit Site" %}</a>
```
Comment (alxngyn) [inline streamwebs_frontend/streamwebs/templates/streamwebs/site_detail.html]: edit site should be shown if the user is 

- logged in 
- has the admin persmissions to edit

Comment (baileysingleton) [inline streamwebs_frontend/streamwebs/templates/streamwebs/site_detail.html]: Fixed!

**Comment by alxngyn** [comment]
i like the flexability of moving the info around.
I don't think there is the most "right" way of doing this.
++ for me once the merge conflict is resolved.

**Pull request merged**